java/example/calypso/android/nfc/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Fri Feb 14 14:43:59 CET 2020
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```

Comment:
we have deployed 5.6.4 on the docker image, can it be 5.6.4  instead of 5.4.1?

Reply:
Android Studio has a fresh update supporting v5.6.4 (late feb 2020). I'll update and test.

Reply:
Done